map/src/components/Map/Map.js

```diff
@@ -886,30 +889,30 @@ const Map = ({data}) => {
             });
 
         });
-    
-    }, [])
+
+    }, [regionsIndex, repIndex])
```

Comment:
Updated this because the we should rerun this effect if `regionsIndex` or `repIndex` changes